source/form.html

```diff
@@ -0,0 +1,151 @@
+      </nav>
+    </div>
+  </header>
+  <section class="programs">
+    <h1 class="visually-hidden">Форма для заполнения данных</h1>
+    <h2 class="programs__title">Подбор программы</h2>
+    <p class="programs__description">Заполните анкету, и мы подберем программу питания для вашего кота</p>
+    <div class="programs__field-name">
+      <label class="programs__label" for="programs-name">Имя:*</label>
+      <input class="programs__input" id="programs-name" name="programs-name" placeholder="Барсик" required>
+    </div>
+    <div class="programs__field-weight">
+      <label class="programs__label" for="programs-weight">Вес (кг):*</label>
+      <input class="programs__input" id="programs-weight" name="programs-weight" placeholder="7" required>
+    </div>
+    <div class="programs__field-age">
+      <label class="programs__label" for="programs-age">Вес (кг):*</label>
+      <input class="programs__input" id="programs-age" name="programs-age" placeholder="7" required>
+    </div>
+    <ul class="programs__radiobuttons__list">
+      <li class="programs__radiobuttons__item">
+        <input class="radiobutton-slim" type="radio" id="slim" name="slim" value="">
+        <label for="slim">Похудение</label>
+      </li>
+      <li class="programs__radiobuttons__item">
+        <input class="radiobutton-mass" type="radio" id="mass" name="mass" value="">
+        <label for="mass">Набор массы</label>
+      </li>
+      <li class="programs__radiobuttons__item">
+        <input class="radiobutton-uncertain" type="radio" id="uncertain" name="uncertain" value="">
+        <label for="uncertain">Не знаю (Нужен ваш совет)</label>
+      </li>
+    </ul>
+  </section>
+  <section class="feedback">
+    <h2 class="visually-hidden">Форма обратной связи</h2>
+    <p class="feedback-title">Контактные данные (владельца кота)</p>
+    <div class="feedback__field-email">
+      <label class="feedback__label" for="feedback-email">E-mail:*</label>
+      <input class="feedback__input" id="feedback-email" name="feedback-email" placeholder="kuklachev@gmail.com"
+        required>
+    </div>
+    <div class="feedback__field-phone">
+      <label class="feedback__label" for="feedback-phone">Телефон:*</label>
+      <input class="feedback__input" id="feedback-phone" name="feedback-phone" placeholder="8 (960) 900-60-90" required>
+    </div>
+    <p class="feedback-title">Комментарий</p>
+    <p><textarea name="comment"></textarea></p>
+    <p class="feedback-title">Дополнительно</p>
+    <ul class="feedback__checkbox__list">
+      <li class="feedback__checkbox__item">
+        <input class="feedback__input" type="checkbox" id="feedback__label" name="feedback__label" value="">
+        <label for="feedback__label">Сахарозаменитель</label>
+      </li>
+      <li class="feedback__checkbox__item">
+        <input class="feedback__input" type="checkbox" id="feedback__label" name="feedback__label" value="">
+        <label for="feedback__label">Питьевая вода</label>
```

Comment:
label class="feedback__label" for="sugar"

input class="feedback__input" type="checkbox" id="sugar" name="sugar" value="sugar"